### PR TITLE
Resurrect structured data on AMP pages

### PR DESCRIPTION
**Pull request by MatthewJWalls**
## What does this change?

(readers note: Structured Data is a simple `<script>{json structure}</script>` tag that must appear in the `<head>` of a page)

A long time ago, all Live Blog structured data code lived inside the Common module, because I thought that the only way to provide custom content inside of `<head>` was from within Common.

I then realised that the main html template had a parameter that lets you pass in custom html that gets rendered inside of head. This meant we could take all the structured data code for live blogs out of Common, and put it back into Article where it rightly belongs, and pass the structured data as custom html into the template's head parameter. All was good.

However when we did this, we neglected to apply the new logic universally. We presented structured data on the normal live blog pages, but *not* on the AMP live blog version.

This PR puts the Structured Data definitions into the AMP version of the Live Blog articles.

One catch: it looks like the mainAmp template doesn't support passing in custom Html into the head like the normal main template does. So I have ADDED support for this by introducing the new parameter.

## What is the value of this and can you measure success?

Getting structured data working again on AMP pages will make the Google Indexing API work

### CODE REVIEW

File: article/app/views/articleAMP.scala.html [hunk 1]
```diff
@@ -1,5 +1,7 @@
 @(model: ArticlePage)(implicit request: RequestHeader, context: _root_.model.ApplicationContext)
 
 @mainAMP(model, model.related, model.article.content){
+
+}{
```
Comment (TBonnin) [inline article/app/views/articleAMP.scala.html]: Amp article does not get structured data?

File: article/app/views/liveBlogAMP.scala.html [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
 @(model: LiveBlogPage)(implicit request: RequestHeader, context: _root_.model.ApplicationContext)
 
 @mainAMP(model, model.related, model.article.content){
+    <script type="application/ld+json">
+        @Html(Organisation().toString())
+    </script>
+    <script type="application/ld+json">
+        @Html(LiveBlogPosting(
+            model.article,
+            LiveBlogHelpers.blocksForLiveBlogRequest(model.article, request.getQueryString("page"))
```
Comment (TBonnin) [inline article/app/views/liveBlogAMP.scala.html]: Unrelated but would it make more sense to only pass `model.article` and `page` request param to the `LiveBlogPosting` constructor and let it  get the blocks internally rather than exposing this `LiveBlogHelpers.blocksForLiveBlogRequest` outside of the `LiveBlogPosting` class?

**Review by TBonnin — APPROVED** [review]
lgtm.
And thanks for this great PR message that describes the context and explains what this patch does in plain english 🥇

File: article/app/views/articleAMP.scala.html (hunk 1 above)
Comment (MatthewJWalls) [inline article/app/views/articleAMP.scala.html]: non-liveblog articles do not receive any structured data so we pass through nothing in this case.

Comment (TBonnin) [inline article/app/views/articleAMP.scala.html]: Got it.
